README.md

```diff
@@ -1320,6 +1321,12 @@ verifier-client \
     --verbose diag
 ```
 
+#### --interface \<device\>
+
+Initiate connections from the specified interface. You can enter any name supported by setsockopt BINDTODEVICE, such at eth0:1.
```

Comment:
Oh, let's not mention the socket option BINDTODEVICE now that we just manually bind. Maybe simplify it to just:

`Initiate connections from the specified interface, such as eth0:1.`